api-channel-jira/src/main/java/com/blackduck/integration/alert/api/channel/jira/lifecycle/JiraSchedulingManager.java

```diff
@@ -0,0 +1,59 @@
+package com.blackduck.integration.alert.api.channel.jira.lifecycle;
+
+import com.blackduck.integration.alert.api.task.TaskManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class JiraSchedulingManager {
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private final TaskManager taskManager;
+
+    @Autowired
+    public JiraSchedulingManager(TaskManager taskManager) {
+        this.taskManager = taskManager;
+    }
+
+    public List<JiraTask> scheduleTasks(List<JiraTask> tasks) {
+        List<JiraTask> acceptedTasks = new ArrayList<>();
+        for (JiraTask task : tasks) {
+            logger.debug("Perform scheduling jira tasks for config with id {} and name {}", task.getConfigId(), task.getConfigName());
+            unscheduleTasks(task.getConfigId());
```

Comment:
It's picky, but can we switch lines 26 and 25? IMO the order or logging messages could be misleading to a customer.

Reply:
Will make the change